.circleci/workflows.yml

```diff
@@ -1913,7 +1914,7 @@ jobs:
       - restore_cached_workspace
       - run:
           name: Build
-          command: yarn workspace @cypress/webpack-preprocessor build
+          command: yarn lerna run build --scope @cypress/webpack-preprocessor
```

Comment:
In order for these jobs to leverage caching we need to change them from `yarn workspace` commands to `yarn lerna run` commands